recipes/hunspell/all/conanfile.py

```diff
@@ -25,41 +28,69 @@ class HunspellConan(ConanFile):
         "shared": False,
         "fPIC": True,
     }
-    generators = "cmake"
-    # FIXME: Remove once the pending upstream PR for CMake support is merged
-    exports_sources = "CMakeLists.txt"
-    no_copy_source = True
+
+    @property
+    def _minimum_cpp_standard(self):
+        return 11
+
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "gcc": "7",
+        }
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            check_min_cppstd(self, self._minimum_cpp_standard)
+        min_version = self._minimum_compilers_version.get(str(self.settings.compiler))
+        if min_version:
+            if Version(self.settings.compiler.version) < min_version:
+                raise ConanInvalidConfiguration(f"{self.name} requires at least {self.settings.compiler} {min_version}")
+
+    def export_sources(self):
+        # FIXME: Remove once the pending upstream PR for CMake support is merged
```

Comment:
```suggestion
        # TODO: Remove once PR is merged: https://github.com/hunspell/hunspell/pull/704/
```
FIXME and XXX are for something that is not working. TBH I don't think that PR will be merged but let's point here.